spring-cloud-kubernetes-fabric8-config/src/main/java/org/springframework/cloud/kubernetes/fabric8/config/reload/Fabric8EventBasedSecretsChangeDetector.java

```diff
@@ -124,16 +124,19 @@ private SecretInformerAwareEventHandler(SharedIndexInformer<Secret> informer) {
 
 		@Override
 		public void onAdd(Secret secret) {
+			log.debug("Secret " + secret.getMetadata().getName() + " was added.");
 			onEvent(secret);
 		}
 
 		@Override
 		public void onUpdate(Secret oldSecret, Secret newSecret) {
+			log.debug("Secret " + newSecret.getMetadata().getName() + " was updated.");
 			onEvent(newSecret);
 		}
 
 		@Override
 		public void onDelete(Secret secret, boolean deletedFinalStateUnknown) {
+			log.debug("Secret " + secret.getMetadata().getName() + " was deleted.");
```

Comment:
log everywhere in debug mode, otherwise the logs are going to get pretty polluted